CONTRIBUTING.md

```diff
@@ -259,37 +342,36 @@ The project contains [.editorconfig](https://github.com/chocolatey-community/cho
 
 Keep the package source files clean and remove obsolete or outdated code and unnecessary comments. Comment non-obvious code so that others can easily understand what it does.
 
-# 3. Teamwork
+### Update Script
+
+An update script are the PowerShell scripts responsible for getting the information about what the most current version of the software is, where it can be acquired from and is typically called `update.ps1`.
```

Comment:
```suggestion
Update scripts are the PowerShell scripts responsible for getting the information about what the most current version of the software is, where it can be acquired from, and how to modify the source code to produce the package. It is typically called `update.ps1`.
```